src/components/AccessibilityBody/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
```

Comment:
Just FYI, we now have a [`Hyperlink` `Paragon` component](https://github.com/edx/paragon/blob/master/src/Hyperlink/index.jsx)